highfidelity/hifi
# Fix Avatar apps warnings.

Added notification for HMD.active property to make sure value will be changed

Test plan:
- run interface
- open Avatar app
- check logs for warnings, related to avatar app

https://highfidelity.fogbugz.com/f/cases/20357/AvatarApp-causes-QML-warnings-in-logs
